test_priority_queue.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+
+# .insert(item): inserts an item into the queue.
+def test_insert():
+    """Test if value is inserted into the queue."""
+    from priority_queue import PriorityQueue
+    pq = PriorityQueue()
+    item = (1, 'value')
+    pq.insert(item)
+    assert pq.container[0] == (1, 'value')
```

Comment:
There isn't much being tested here. Try and thing of more ways to test that insert is working correctly, and make sure to cover edge cases.